stacks/e4s-rocm-external/spack.yaml

```
@@ -239,6 +239,7 @@ spack:
   - hpx +rocm amdgpu_target=gfx908
   - hypre +rocm amdgpu_target=gfx908
   - kokkos +rocm amdgpu_target=gfx908
+  - kokkos-fft device_backend=hipfft ^kokkos +rocm amdgpu_target=gfx908
```

Should there be a `+tests` here?
```suggestion
  - kokkos-fft device_backend=hipfft +tests ^kokkos +rocm amdgpu_target=gfx908
```

---

Yes indeed thx